indico/modules/events/registration/controllers/display.py

```diff
@@ -110,20 +111,25 @@ def _process(self):
                        ~RegistrationForm.is_deleted,
                        ~Registration.is_deleted,
                        _join=Registration.registration_form)
-                 .order_by(db.func.lower(Registration.last_name), db.func.lower(Registration.first_name)))
-        registrations = [(reg.get_full_name(), reg.get_personal_data()) for reg in query]
+                 .order_by(*Registration.order_by_name)
+                 .options(contains_eager('registration_form')))
```

Comment:
you can also use `_eager` here like in the other query (and then remove the contains_eager option/import)